scripts/emulator.js

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env node
+
+// inspired by `np` – https://github.com/sindresorhus/np
+
+const Listr = require('listr')
+const inquirer = require('inquirer')
+
+const { execSync } = require('child_process')
+
+const { add } = require('rambdax')
+
+const emulators = execSync(`$ANDROID_HOME/emulator/emulator -list-avds`).toString()
+const sdks = execSync(
+  `$ANDROID_HOME/tools/bin/sdkmanager --list | grep "system-images/" `,
+).toString()
+
+const askForEmu = [
+  {
+    type: 'list',
+    name: 'name',
+    message: 'Pick Emulator from list or add a new one',
+    pageSize: add(emulators.length, 4),
```

Comment:
lol :P `emulator.length + 4` is fine too :P

Reply:
I was inspired by https://github.com/Nozbe/WatermelonDB/blob/master/scripts/publish.js#L36 😆